_source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md

```diff
@@ -72,8 +85,8 @@ You can click the <i class="fas fa-signal"></i> icon for a top-N analysis of the
 
 
 ### Split and Sync
-
-You can split your Explore mode screen to work faster and make quick comparisons. You can split the screen to work with two views side-by-side, whether Metrics-Metrics, Logs-Logs, or Logs-Metrics.
+<!-- still relevant-->
```

Comment:
Right - Still relevant.
It's especially effective for comparing queries from different time ranges or event different data sources (in Logz.io data source = metrics account).